### PR TITLE
fix(create-injection-token): add injector to create injection token

### PR DESCRIPTION
**Pull request by nartc**

### CODE REVIEW

File: migrations.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+		{
+			"cli": "nx",
+			"version": "16.8.2-beta.0",
+			"description": "Remove invalid options (strict, noInterop) for ES6 type modules.",
+			"factory": "./src/migrations/update-16-8-2/update-swcrc",
+			"package": "@nx/js",
+			"name": "16-8-2-update-swcrc"
+		}
+	]
+}
```
Comment (tomalaforge) [inline migrations.json:20]: This file doesn't need to be commited, doest it ?

Comment (tomer953) [reply migrations.json:20]: Yep, After the migrations have been run, nx no longer needs this file

**Review by eneajaho — APPROVED** [review]
LGTM

File: migrations.json (hunk 1 above)
Comment (eneajaho) [reply migrations.json:20]: Yeah it needs to be removed!

**Pull request merged**